source/assets/javascripts/app/_main.js

```diff
@@ -1,3 +1,18 @@
+$(function gdprBanner() {
+  if (!docCookies.hasItem("gdpr-accept")) {
+    var gdpr = $("<div class='gdpr-cookie-banner'>");
+    gdpr.on("click", ".gdpr-close", function(e) {
+      docCookies.setItem("gdpr-accept", "yes");
+      $(document.body).removeClass("with-gdpr");
+      gdpr.remove();
+    });
+
+    gdpr.append("<i class='gdpr-close fa fa-close'/>").append("<p>Our site uses <a href='https://www.thoughtworks.com/privacy-policy#cookie-section' target='_blank'>cookies</a> for analytics. If you're unsure about it, take a look at our <a href='https://www.thoughtworks.com/privacy-policy' target='_blank'>privacy policy</a>.</p>");
```

Comment:
See https://www.jitbit.com/alexblog/256-targetblank---the-most-underestimated-vulnerability-ever/ for `target=blank` vulnerability and how to avoid it.

Reply:
@ketan thanks for the link - fixed!